DEV_GUIDE.md

```diff
@@ -0,0 +1,141 @@
+# PlayerUI iOS Development Guide
+
+### Sections:
+1. [Setup](#Setup)
+2. [Adding a new plugin](#adding-a-new-plugin)
+
+## Setup
+### Demo Application
+The demo app can be built and launched from the command line with bazel:
+```bash
+bazel build //docs/site:site
+bazel run //docs/site:site
+```
+
+## Adding a new plugin
+Adding a new plugin is simple using predefined macros for `BUILD.bazel` files. Additional steps are required to include new code in the final artifacts, and in the Xcode project.
```

Comment:
Xcode probably not relevant for this doc

Reply:
Oops, copied the iOS doc over. Forgot to get rid of this..